content/docs/Index-Structure.md

```diff
@@ -0,0 +1,130 @@
++++
+author = ["Patrick Mézard"]
+date = "2015-10-09T00:00:00+01:00"
+title = "Index Structure"
+[menu.docs]
+weight = 30
+parent = 'devguide'
+
++++
+
+Bleve default index, called "upside_down" is stored in a single key/value store
+table. The store must be able to enumerate its entries starting at a given key,
+in lexicographic byte order.
+
+Index key and values are handled as byte arrays and are called rows (see `index/upside_down/row.go` for details). To store different row types in a single table, bleve prefixes their keys with a single byte, for instance the term frequency keys start with a 'f'. The following sections describe the data structures stored in the index with pseudo-Go code for value layout.
```

Comment:
term frequencies rows are prefixed with 't' not 'f'.  field definition rows are prefixed with 'f'.